packages/lambda-tiler/src/routes/tile.ts

```diff
@@ -164,6 +164,7 @@ export async function wmts(req: LambdaContext): Promise<LambdaHttpResponse> {
     response.header(HttpHeader.ETag, cacheKey);
     response.header(HttpHeader.CacheControl, 'max-age=0');
     response.buffer(data, 'text/xml');
+    req.set('bytes', data.byteLength);
```

Comment:
Does it suppose to set to response?

Reply:
This `set` set's the request context.

The request context is logged at the end of the request.

Reply:
I see. Now good to go.